src/networking.c

```diff
@@ -521,6 +521,17 @@ void afterErrorReply(client *c, const char *s, size_t len, int flags) {
             showLatestBacklog();
         }
         server.stat_unexpected_error_replies++;
+
+        /* If this command was from our master, we are not a writable replica,
+         * and replication resync on error is enabled, disconnect and try to 
+         * get a fresh copy of data. */
+        if (ctype == CLIENT_TYPE_MASTER && server.repl_slave_ro &&
```

Comment:
Also check for AOF clients.

Reply:
i also think we should allow a mode where we panic both for replica and AOF clients

Reply:
For some reason I thought Yossi said it already paniced, but upon checking it does not, so yes this needs to still be addressed.